hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -1563,7 +1563,7 @@
         <partition-lost-listeners>
             <partition-lost-listener>com.your-package.YourPartitionLostListener</partition-lost-listener>
         </partition-lost-listeners>
-        <merge-policy>com.hazelcast.cache.merge.LatestAccessCacheMergePolicy</merge-policy>
+        <merge-policy batch-size="100">LatestAccessMergePolicy</merge-policy>
```

Comment:
I guess this batch-size is not yet parsed, right? Maybe you can create a new issue so we don't forget.

Reply:
fixed